here-naksha-app-service/src/test/java/com/here/naksha/app/service/CreateFeatureTestHelper.java

```diff
@@ -130,27 +125,24 @@ public void tc0300_testCreateFeaturesWithNewIds() throws Exception {
     assertEquals(200, response.statusCode(), "ResCode mismatch. Failed creating Storage");
 
     // Given: EventHandler (uses above Storage) configured in Admin storage
-    final EventHandler eventHandler =
-        parseJsonFileOrFail("TC0300_createFeaturesWithNewIds/create_event_handler.json", EventHandler.class);
-    final WriteFeatures<?> ehRequest = createFeatureRequest(
-        NakshaAdminCollection.EVENT_HANDLERS, eventHandler, IfExists.REPLACE, IfConflict.REPLACE);
-    try (final IWriteSession writer =
-        app.getHub().getSpaceStorage().newWriteSession(newTestNakshaContext(), true)) {
-      final Result result = writer.execute(ehRequest);
-      assertTrue(result instanceof SuccessResult, "Failed creating EventHandler");
-      writer.commit();
-    }
+    final String eventHandlerJson = loadFileOrFail("TC0300_createFeaturesWithNewIds/create_event_handler.json");
+    request = HttpRequest.newBuilder(stdHttpRequest, (k, v) -> true)
+        .uri(new URI(nakshaHttpUri + "hub/handlers"))
+        .POST(HttpRequest.BodyPublishers.ofString(eventHandlerJson))
+        .header(HDR_STREAM_ID, streamId)
+        .build();
+    response = httpClient.send(request, HttpResponse.BodyHandlers.ofString());
+    assertEquals(200, response.statusCode(), "ResCode mismatch. Failed creating Event Handler");
 
     // Given: Space (uses above EventHandler) configured in Admin storage
     final Space space = parseJsonFileOrFail("TC0300_createFeaturesWithNewIds/create_space.json", Space.class);
```

Comment:
Perhaps it is better to use two functions here?
1. use ```TestUtil.loadFileOrFail()``` which will return Space as JSON string, which we should pass in Http request (instead of ```space.toString()``` )
2. use ```TestUtil.parseJson()``` which can convert JSON string into Space object, which is also later required for calling Feature API

Reply:
Ok I will add that tomorrow, to make sure we have a JSON string without relying on .toString()